pkg/api/api.go

```diff
@@ -166,10 +166,12 @@ func (a *API) RegisterAlertmanager(am *alertmanager.MultitenantAlertmanager, tar
 }
 
 // RegisterAPI registers the standard endpoints associated with a running Cortex.
-func (a *API) RegisterAPI(httpPathPrefix string, cfg interface{}) {
+func (a *API) RegisterAPI(httpPathPrefix string, actualCfg interface{}, defaultCfg interface{}) {
 	a.indexPage.AddLink(SectionAdminEndpoints, "/config", "Current Config")
 
-	a.RegisterRoute("/config", configHandler(cfg), false, "GET")
+	a.RegisterRoute("/config", configHandler(actualCfg), false, "GET")
+	a.RegisterRoute("/config/defaults", configHandler(defaultCfg), false, "GET")
+	a.RegisterRoute("/config/diff", configDiffHandler(actualCfg, defaultCfg), false, "GET")
```

Comment:
Having three routes for config seems like overkill. I think single `/config` is just fine (we're also adding `/runtime_config` in another PR), with a `diff` param switching to diff view. Also I'd suggest to include it in the index page.

Reply:
Feedback I got from github.com/leventebalogh (who is working on Config UI) was that it would be helpful to have just the default values as well.

Does it make sense to use `mode=defaults`/`mode=diff` instead?

Reply:
> Does it make sense to use `mode=defaults`/`mode=diff` instead?

👍 Sure, thanks!